lib/sablon/image.rb

```diff
@@ -0,0 +1,17 @@
+module Sablon
+  class Image
+    include Singleton
+    attr_reader :definitions
+
+    Definition = Struct.new(:name, :data, :rid) do
+      def inspect
+        "#<Image #{name}:#{data}:#{rid}"
+      end
+    end
+
+    def self.create_by_path(path, random = nil)
```

Comment:
Could we name the second argument `name` and then make it possible to specify the full name instead of just the random prefix? This would allow a user to name the images in the zip.